.circleci/config.yml

```diff
@@ -581,6 +581,24 @@ jobs:
           path: artifacts
           destination: artifacts
           when: always
+      - run:
+          name: Zip packs from GCS
+          command: |
+            if [ $CIRCLE_NODE_INDEX -ne 0 ] ;
+              then
+                echo 'Skipping - running only in container number 0.'
+                exit 0;
+            fi
+            GCS_PATH=$(mktemp)
+            ZIP_FOLDER=$(mktemp -d)
+            echo $GCS_MARKET_KEY > $GCS_PATH
+            python3 ./Tests/Marketplace/zip_packs.py -b 'marketplace-ci-build' -z $ZIP_FOLDER -a $CIRCLE_ARTIFACTS -s $GCS_PATH -n $CIRCLE_BUILD_NUM -br $CIRCLE_BRANCH
```

Comment:
What about production bucket `marketplace-dist` that runs on master? Don't we need this zip also in master?